aiida_common_workflows/workflows/relax/vasp/workchain.py

```diff
@@ -15,7 +15,9 @@
 def get_stress(stress):
     """Return the final stress array."""
     stress_data = orm.ArrayData()
-    stress_data.set_array(name='stress', array=stress.get_array('final'))
+    stress_kbar = stress.get_array('final')
+    stress_ev_per_angstr3 = stress_kbar / 1602.1766208
```

Comment:
You can also use [`pint`](https://pint.readthedocs.io/en/stable/index.html) for the unit conversions. Some example usage:

```python
In [1]: from pint import UnitRegistry; ureg = UnitRegistry()

In [2]: stress = 1 * ureg.kbar

In [3]: stress.to('eV / angstrom^3')
Out[3]: 0.0006241509074460764 <Unit('electron_volt / angstrom ** 3')>
```